src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraSchema.scala

```diff
@@ -1,140 +0,0 @@
-package com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra
-
-import java.time.Instant.now
-import java.util.UUID
-
-import com.microsoft.partnercatalyst.fortis.spark.dto.FortisEvent
-import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.Utils.{mean, rescale}
-import com.microsoft.partnercatalyst.fortis.spark.transforms.gender.GenderDetector.{Female, Male}
-import com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment.SentimentDetector.Neutral
-
-case class Sentiment(
-  pos_avg: Float,
-  neg_avg: Float)
-
-case class Gender(
-  male_mentions: Int,
-  female_mentions: Int)
-
-case class Entities(
-  name: String,
-  externalsource: String,
-  externalrefid: String,
-  count: Float)
-
-case class Features(
-  mentions: Int,
-  sentiment: Sentiment,
-  gender: Gender,
-  entities: Set[Entities])
-
-case class Event(
-  pipeline: String,
-  externalid: String,
-  computedfeatures: Features,
-  detectedkeywords: Set[String],
-  detectedplaceids: Set[String],
-  event_time: Long,
-  eventlangcode: String,
-  id: UUID,
-  insertion_time: Long,
-  messagebody: String,
-  sourceid: String,
-  sourceurl: String,
-  title: String)
-
-case class SiteSetting(
-  id: UUID,
-  sitename: String,
-  geofence: Seq[Double],
-  languages: Set[String],
-  defaultzoom: Int,
-  title: String,
-  logo: String,
-  translationsvctoken: String,
-  cogspeechsvctoken: String,
-  cogvisionsvctoken: String,
-  cogtextsvctoken: String,
-  insertion_time: Long
-)
-
-case class Stream(
```

Comment:
@jcjimenez I'll add them elsewhere as you've proposed once they're needed. Thanks for pointing this out.